features/admin.connections.v1/components/create/custom-authentication-create-wizard.tsx

```diff
@@ -0,0 +1,887 @@
+                                "customAuthentication:fields.createWizard.configurationsStep." +
+                                    "authenticationTypeDropdown.authProperties.header.label"
+                            ) }
+                            placeholder={ t(
+                                "customAuthentication:fields.createWizard.configurationsStep." +
+                                    "authenticationTypeDropdown.authProperties.header.placeholder"
+                            ) }
+                            required={ true }
+                            maxLength={ 100 }
+                            minLength={ 0 }
+                            data-componentid={ `${_componentId}-authentication-property-header` }
+                            width={ 15 }
+                        />
+                        <Field.Input
+                            ariaLabel="value"
+                            className="addon-field-wrapper"
+                            name="valueAuthProperty"
+                            inputType="password"
+                            type={ showSecondarySecret ? "text" : "password" }
+                            InputProps={ {
+                                endAdornment: renderInputAdornmentOfSecret(showSecondarySecret, () =>
+                                    setShowSecondarySecret(!showSecondarySecret)
+                                )
+                            } }
+                            label={ t(
+                                "customAuthentication:fields.createWizard.configurationsStep." +
+                                    "authenticationTypeDropdown.authProperties.value.label"
+                            ) }
+                            placeholder={ t(
+                                "customAuthentication:fields.createWizard.configurationsStep." +
+                                    "authenticationTypeDropdown.authProperties.value.placeholder"
+                            ) }
+                            required={ true }
+                            maxLength={ 100 }
+                            minLength={ 0 }
+                            data-componentid={ `${_componentId}-authentication-property-value` }
+                            width={ 15 }
+                        />
+                    </>
+                );
+            default:
+                break;
+        }
+    };
+
+    /**
+     * This method validates the endpoint configurations.
+     * @param values - values to be validated.
+     * @returns errors object.
+     */
+    const validateEndpointConfigs = (
+        values: EndpointConfigFormPropertyInterface
+    ): Partial<EndpointConfigFormPropertyInterface> => {
+        const errors: Partial<EndpointConfigFormPropertyInterface> = {};
+
+        if (!values?.endpointUri) {
+            errors.endpointUri = t(
+                "customAuthentication:fields.createWizard.configurationsStep." + "endpoint.validations.empty"
```

Comment:
```suggestion
                "customAuthentication:fields.createWizard.configurationsStep.endpoint.validations.empty"
```
Let's remove these unnecessary concatenations. Please check other places as well.